### PR TITLE
feat(er-next): progress module in navigation bar

### PR DESCRIPTION
**Pull request by DrShpongle**
This adds progress module in navigation bar:

<details>
  <summary>Click to preview</summary>
  
</details>

Black And White Thinking GIF by Tobias Rothe

### CODE REVIEW

File: apps/epic-react/src/templates/lesson-template.tsx [hunk 1]
```diff
@@ -1,44 +1,91 @@
...
+import Spinner from '@/components/spinner'
+import Layout from '@/components/app/layout'
+import ProgressBar from '@/components/progress-bar'
+import {getOgImage} from '@/utils/get-og-image'
 import {track} from '@/utils/analytics'
 import {lessonPathBuilder} from '@/utils/lesson-path-builder'
 
+const NavigationProgressModule: React.FC<{
+  module: Module
+  moduleProgress: any
```
[the remark this answers is not in the document]
Comment (zacjones93) [reply apps/epic-react/src/templates/lesson-template.tsx:45]: @DrShpongle this looks like the type you are looking for? 

https://github.com/skillrecordings/products/blob/main/packages/skill-lesson/video/module-progress.tsx#L29-L75

File: apps/epic-react/src/templates/lesson-template.tsx [hunk 2]
```diff
@@ -333,6 +393,7 @@ const ExerciseTemplate: React.FC<{
                     >
                       <LessonList
                         module={module}
```
Comment (zacjones93) [inline apps/epic-react/src/templates/lesson-template.tsx:395]:
```suggestion
                        module={epicReactModule}
```
I must have missed this with the merge conflicts I was getting but we want to pass `epicReactModule` so the `moduleType` is always 'module'

File: apps/epic-react/src/templates/lesson-template.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (DrShpongle) [reply apps/epic-react/src/templates/lesson-template.tsx:45]: @zacjones93 exactly, thanks! Weird I missed it when searched 🤔

File: apps/epic-react/src/templates/lesson-template.tsx (hunk 2 above)
Comment (DrShpongle) [reply apps/epic-react/src/templates/lesson-template.tsx]: Fixed

**Review by zacjones93 — APPROVED** [review]

**Pull request merged**